index.d.ts

```diff
@@ -2328,6 +2332,8 @@ type AlignItemsProperty = Globals | BaselinePosition | SelfPosition | "normal" |
 
 type AlignSelfProperty = Globals | "auto" | "baseline" | "center" | "flex-end" | "flex-start" | "stretch";
 
+type AllProperty = Globals | "inherit" | "initial" | "revert" | "unset";
```

Comment:
Not a big deal, but isn’t this redundant?

Reply:
Yeah.

Reply:
I finally found a [confirmation](https://www.w3.org/TR/2016/CR-css-cascade-4-20160114/#all-shorthand) that `all` has a direct relationship with the CSS-wide keywords.

> It only accepts the CSS-wide keywords.

So it should be `type AllProperty = Globals;`